src/berry_mill/mill.py

```diff
@@ -99,6 +136,19 @@ def _init_local_repos(self) -> None:
                 self.cfg.raw_unsafe_config()["repos"]["local"][arch].update(jr[arch])
         return
 
+    def _construct_final_appliance(self) -> None:
+        """
+        Constructs final kiwi appliance which is used by kiwi
+        1. moves the appliance description to a tmp dir, so kiwi wont use this "wrong" one
+        2. Constructs the right appliance and safes it named as the one passed to berrymill orginially
+        """
+        final_rendered_xml_string = Loader().load(self._appliance_abspath)
+        
+        shutil.move(self._appliance_abspath, self._bac_appliance_abspth)
+
+        with open(self._appliance_abspath, "w") as ma:
```

Comment:
Maybe remove these empty lines?
We need MyPy and autoformatter, after all. 😉

Reply:
That is a PR for another time

Reply:
Then I can do everything at once